frontend/src/components/Items.vue

```diff
@@ -127,13 +142,43 @@ export default {
       },
       immediate: true,
     },
+    items: {
+      handler: function () {
+        if (
+          this.items.some((item) => item.hasOwnProperty("activeState")) &&
+          this.mode === 'single'
+        ) {
+          let activeItem = null;
+          for (let i = 0; i < this.items.length; i++) {
```

Comment:
elsewhere in the code you implemented a **deep** search for items/subItems.
Is a flat search sufficient here?